content/en/docs/concepts/storage/volumes.md

```diff
@@ -543,6 +543,63 @@ spec:
       type: FileOrCreate
 ```
 
+### image
+
+{{< feature-state feature_gate_name="ImageVolume" >}}
+
+An `image` volume source represents an OCI object (a container image or
+artifact) pulled and mounted on the kubelet's host machine.
+
+One example to use the `image` volume source is:
+
+{{% code_sample file="pods/image-volumes.yaml" %}}
+
+The volume is resolved at pod startup depending on which `pullPolicy` value is
+provided:
+
+- `Always`: the kubelet always attempts to pull the reference. Container
+  creation will fail If the pull fails.
+- `Never`: the kubelet never pulls the reference and only uses a local image or
+  artifact. Container creation will fail if the reference isn't present.
+- `IfNotPresent`: the kubelet pulls if the reference isn't already present on
+  disk. Container creation will fail if the reference isn't present and the pull
+  fails.
+
+The volume gets re-resolved if the pod gets deleted and recreated, which means
+that new remote content will become available on pod recreation. A failure to
+resolve or pull the image during pod startup will block containers from starting
+and may add significant latency. Failures will be retried using normal volume
+backoff and will be reported on the pod reason and message. The types of objects
+that may be mounted by this volume are defined by the container runtime
+implementation on a host machine and at minimum must include all valid types
+supported by the container image field. The OCI object gets mounted in a single
+directory (`spec.containers[*].volumeMounts.mountPath`) by merging the manifest
+layers in the same way as for container images. The volume will be mounted
+read-only (`ro`) and non-executable files (`noexec`). Sub path mounts for
+containers are not supported (`spec.containers[*].volumeMounts.subpath`). The
+field `spec.securityContext.fsGroupChangePolicy` has no effect on this volume
+type. The [`AlwaysPullImages` Admision Controller](/docs/reference/access-authn-authz/admission-controllers/#alwayspullimages)
+does also work for this volume source like for container images.
+
+The following fields are available for the `image` type:
+
+- `reference`: Image or artifact reference to be used.
+    Behaves in the same way as `pod.spec.containers[*].image`.
+    Pull secrets will be assembled in the same way as for the container image by
+    looking up node credentials, SA image pull secrets, and pod spec image pull
```

Comment:
What is "SA" here? Would it make sense to make a glossary pop up for this?

Reply:
(it's a _Service Account_, of course; no need to answer :smile: )

Reply:
I'll write it out, thanks!